### ISSUE TITLE
[bug][v2][macos] Can't build Android - vanilla install

### ISSUE THREAD

**Issue by oddpxl** [tauri-apps/tauri#10065]
### Describe the bug

1 - Clean install
2 - Changing tauri.conf.json identifier --> ""identifier": "com.myname.dev"
3 - Deleting gen/android folder and running "tauri android init"
4 - pnpm tauri android build

```
Error Project directory .../tauri-app/src-tauri/gen/android/app/src/main/java/com/myname/dev does not exist. Did you update the package name in `Cargo.toml` or the bundle identifier in `tauri.conf.json > identifier`? Save your changes, delete the `gen/android` folder and run `tauri android init` to recreate the Android project.
 ELIFECYCLE  Command failed with exit code 1.

```

### Reproduction

1 - Clean install -->  pnpm create tauri-app --beta
2 - Name tauri-app / TypeScript / JavaScript  / pnpm / Solid / Typescript / Mobile YES 
3 -  cd tauri-app - pnpm install - pnpm tauri android init
4 - Change tauri.conf.json identifier
5 - Delete gen/android folder and run "tauri android init"
6 - pnpm tauri android build

### Expected behavior

It would build as usual.

### Full `tauri info` output

```text
[✔] Environment
    - OS: Mac OS 14.4.1 X64
    ✔ Xcode Command Line Tools: installed
    ✔ rustc: 1.79.0 (129f3b996 2024-06-10)
    ✔ cargo: 1.79.0 (ffa9cf99a 2024-06-03)
    ✔ rustup: 1.27.1 (54dd3d00f 2024-04-24)
    ✔ Rust toolchain: stable-aarch64-apple-darwin (default)
    - node: 21.7.1
    - pnpm: 8.15.5
    - npm: 10.5.1

[-] Packages
    - tauri [RUST]: 2.0.0-beta.22
    - tauri-build [RUST]: 2.0.0-beta.17
    - wry [RUST]: 0.40.1
    - tao [RUST]: 0.28.1
    - tauri-cli [RUST]: 1.5.11
    - @tauri-apps/api : not installed!
    - @tauri-apps/cli [NPM]: 2.0.0-beta.20

[-] App
    - build-type: bundle
    - CSP: unset
    - frontendDist: ../dist
    - devUrl: http://localhost:1420/
    - framework: SolidJS
    - bundler: Vite
```

**Comment by pewsheen** [comment]
Could you try running `pnpm tauri android init` instead of `tauri android init` in step 5?

**Comment by oddpxl** [comment]
Alright so here we go...

1 - clean install
2 - Changing tauri.conf.json identifier --> ""identifier": "com.myname.dev"
3 - pnpm tauri android build

` Error Project directory /tauri-app/src-tauri/gen/android/app/src/main/java/com/myname/dev does not exist. Did you update the package name in `Cargo.toml` or the bundle identifier in `tauri.conf.json > identifier`? Save your changes, delete the `gen/android` folder and run `tauri android init` to recreate the Android project.
 ELIFECYCLE  Command failed with exit code 1.`

( This is where I got confused by the instructions - previously I didn't add "pnpm"  )

4 - Delete gen/android folder 
6 - pnpm tauri android init
5 - pnpm tauri android build

`You must change the `version` in `tauri.conf.json`. The default value `0.0.0` is not allowed for Android package and must be at least 0.0.1.`

6 - Changed version to 0.0.1
7 - pnpm tauri android build

WORKS !

Thanks for helping out - much appreciated - not sure why I stumbled on this but hey...  ..learning !!
